README.md

```diff
@@ -1,25 +1,63 @@
-# Code Guidelines
-For reference, see the below guidelines we use for peer code review at Exodus.
+# Code and Process Guidelines
+
+For reference, see the below guidelines we use for peer code review at Exodus. You can follow this checklist when evaluating a code challenge submission.
+
+## Grading
+
+See points in parentheses before each item. Negative points mean the candidate can only lose points for that item. If a candidate blows your mind in some section (e.g. their tests are comprehensive, or their code is super duper awesome readable), you can give them a bonus point.
+
+> [!IMPORTANT]
+> GRADING (TOTAL SCORE)
+>
+> Minimum: 10
+
+## Code
+
+### Functionality
+
+- [ ] (1) All task requirements are fully implemented
+- [ ] (1) Not obviously broken
+- [ ] (1) No dubious functionality on top of the spec
+- [ ] (1) Not over-engineered, e.g., doesn’t prematurely optimize at the expense of complexity
+
+### Code Quality
+
+- [ ] (2) Code is readable: linted, prettified, and written in a consistent style
+- [ ] (1) Code is self-documenting: semantic variables names, function names describe implemented function
+- [ ] (1) Code is as simple as possible and modular
+  - Small single-purpose functions are preferred to giant functions that do 10 things
+  - Code is logically separated (e.g., no giant utils folders)
+- [ ] (2) Author uses objects, arrays, inheritance, closures, callbacks, promises expertly, and is up to date with the language/technology, e.g., async/- [ ] await > promise chains > callbacks for code after 2019
+- [ ] (1) Tests are included and:
+  - Are readable (intent is clear)
+  - Cover happy path
+  - Cover edge cases
+
+### Documentation
+
+- [ ] (2) Clear instructions for project setup
```

Comment:
2 is a bit too much 🤔. I'm thinking either 1 or -1, @timlanahan ?
```suggestion
- [ ] (1) Clear instructions for project setup
```

Reply:
totally agree...this should be a baseline.  maybe negative point is best (which i think is fair esp if it's in the guidelines).  at the very least someone should be able to write a basic description of how to install deps and run the project.  i don't think that should take too long